cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2027,4 +2027,14 @@ public static final class ArtifactCache {
     public static final String BOSS_THREADS = "artifact.cache.boss.threads";
     public static final String WORKER_THREADS = "artifact.cache.worker.threads";
   }
+
+  public static final class MetadataConsumer {
```

Comment:
Do you need to add these in cdap-default.xml as well?

Reply:
I checked other projects, seems like we don't have default values for these

Reply:
removed this now